solutions/swb-reference/integration-tests/tests/multiStep/awsAccount.test.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AccountService, CreateAccountData } from '@aws/workbench-core-accounts';
+import { Account } from '@aws/workbench-core-accounts/lib/models/accounts/account';
+import { resourceTypeToKey } from '@aws/workbench-core-base';
+import DynamoDBService from '@aws/workbench-core-base/lib/aws/helpers/dynamoDB/dynamoDBService';
+import ClientSession from '../../support/clientSession';
+import { AccountHelper } from '../../support/complex/accountHelper';
+import Setup from '../../support/setup';
+import RandomTextGenerator from '../../support/utils/randomTextGenerator';
+import Settings from '../../support/utils/settings';
+
+describe('multiStep awsAccount integration test', () => {
+  const setup: Setup = new Setup();
+  const settings: Settings = setup.getSettings();
+  let adminSession: ClientSession;
+
+  beforeEach(async () => {
+    // expect.hasAssertions();
+    adminSession = await setup.getDefaultAdminSession();
+  });
+
+  test('it works', async () => {
+    const dynamoDbService = new DynamoDBService({ region: 'us-east-1', table: 'swb-dev-va' });
+    const accountService = new AccountService(dynamoDbService);
+
+    const randomTextGenerator = new RandomTextGenerator(setup.getSettings().get('runId'));
+
+    const createAccountParams: CreateAccountData = {
+      hostingAccountHandlerRoleArn: settings.get('hostingAccountHandlerRoleArn'),
+      awsAccountId: settings.get('hostAwsAccountId'),
+      envMgmtRoleArn: settings.get('envMgmtRoleArn'),
+      name: randomTextGenerator.getFakeText('fakeName'),
+      externalId: randomTextGenerator.getFakeText('fakeExternalId')
+    };
+
+    const hostingAwsAccountId = `${resourceTypeToKey.awsAccount}#${createAccountParams.awsAccountId}`;
+    const query = { key: { name: 'pk', value: hostingAwsAccountId } };
+    const ddbEntries = await dynamoDbService.getPaginatedItems(query);
+    if (ddbEntries.data.length > 0) {
+      const accountId = ddbEntries.data[0].accountId.toString();
+      const awsAccountItemKey = {
+        pk: hostingAwsAccountId,
+        sk: `${resourceTypeToKey.account}#${accountId}`
+      };
+      await dynamoDbService.delete(awsAccountItemKey).execute();
+    }
+
+    const createResponse = await adminSession.resources.accounts.create(createAccountParams, false);
+    expect(createResponse.status).toEqual(201);
+
+    const accountId = createResponse.data.id;
```

Comment:
`expect(createResponse.data.id).toBeTruthy()` (or to be defined, but better Truthy)